pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/rebalance/DefaultRebalancePreChecker.java

```diff
@@ -410,6 +410,11 @@ private RebalancePreCheckerResult checkRebalanceConfig(RebalanceConfig rebalance
       }
     }
 
+    if (tableConfig.getTableType() == TableType.OFFLINE && rebalanceConfig.isForceCommitBeforeMoved()) {
+      pass = false;
+      warnings.add("forceCommitBeforeMoved is set for OFFLINE table, which will be ignored.");
```

Comment:
let's rename this to match the parameter name

Reply:
though, I think adding this to pre-checks is probably overkill. why not just set it to `false` internally and instead add a warning log. we don't have a similar pre-check as this for `includeConsuming` either, do we?

Reply:
Yeah +1, I'm not sure pre-checks is the best place to put this. I'd advocate for either logging a warning and overriding it to `false` internally or returning an error back to the user indicating that the option can't be used with `OFFLINE` tables.